squad/core/models.py

```diff
@@ -563,6 +563,31 @@ def has_extra_metadata(self):
             return False
         return True
 
+    __failures__ = None
+
+    @property
+    def failures(self):
+        if self.__failures__ is None:
+            failures = []
+            limit = self.project.build_confidence_count
+
+            failing = self.tests.filter(result=False).exclude(has_known_issues=True).prefetch_related("metadata", "environment")
+
+            for f in failing:
+                failures.append(f)
```

Comment:
at the end, `failing` and `failures` are the same list as you append every item of the former to the later. you only need one of them.